src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,88 @@
+import './nav.css';
+import logo from '../../asset/image/logo.svg'
+
+const Navigation = () => {
+    return(
+        <header className="nav-header header">
+        <div className="mobile-nav">
+            <input type="checkbox" />
+            <span></span><span></span><span></span>
+            <ul className="menu-nav"> 
+                <li className="no-list"> 
+                    <a href="#" className="btn-link" role="button">
+                        <button className="button-icon text-uppercase nav-icon cp">Products
+                            <i className="bi bi-basket"></i>
+                        </button>
+                    </a>
+                </li>
+                <li className="no-list">
+                    <a href="#" className="btn-link" role="button">
+                        <button className="button-icon text-uppercase nav-icon cp">Cart
+                            <i className="bi bi-cart"></i>
+                        </button>
+                    </a>
+                </li>
+                <li className="no-list"> 
+                    <a href="#" className="btn-link" role="button">
+                        <button className="button-icon text-uppercase nav-icon cp">Wishlist
+                            <i className="bi bi-heart"></i>
+                        </button>
+                    </a>
+                </li>
+                <li className="no-list"> 
+                    <a href="#" className="btn-link td" role="button">
+                        <button className="button-icon text-uppercase nav-icon cp">Logout
+                            <i className="bi bi-door-closed"></i>
+                        </button>
+                    </a>   
+                </li>                 
```

Comment:
As these are repeating element make a component.

Reply:
Ok. Thanks for the suggestion.